packages/uhk-common/src/util/constants.ts

```diff
@@ -2,6 +2,6 @@ export namespace Constants {
     export const AGENT_GITHUB_URL = 'https://github.com/UltimateHackingKeyboard/agent';
     export const AGENT_CONTRIBUTORS_GITHUB_PAGE_URL = 'https://github.com/UltimateHackingKeyboard/agent/graphs/contributors';
     export const AGENT_CONTRIBUTORS_GITHUB_API_URL = 'https://api.github.com/repos/UltimateHackingKeyboard/agent/contributors';
-    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://ultimatehackingkeyboard.com/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk'
+    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://uhk.io/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk';
```

Comment:
I'd rather prefer ultimatehackingkeyboard.com. We only use our uhk.io in onboarding URLs. Using the full URL also makes opening such links faster, as no redirection is involved.

Reply:
That puts the URL over the linter's limit.  I could break this into separate lines?  

But that brings up the question again of do you guys WANT all URLs in the constants file?  It seems kind of weird to have FAQ links embedded in the constants file.  For those, I'd think it better to have them just embedded in the HTML like it was originally.  Especially if you want to add more links in the help page over time.

Reply:
Sure, you can break overly large strings into separate lines.

I agree that it's simpler to embed such URLs in the HTML. Please do.